app/components/primer/alpha/segmented_control.ts

```diff
@@ -0,0 +1,38 @@
+import {controller, targets} from '@github/catalyst'
+
+@controller
+export class SegmentedControlElement extends HTMLElement {
+  @targets buttons: HTMLElement[]
```

Comment:
It doesn't look like this is used anywhere

Reply:
Good catch 👍🏻